Makefile

```diff
@@ -3,9 +3,11 @@ GO := go
 GO_BUILD_PACKAGES := ./cmd/...
 GO_BUILD_BINDIR :=./bin
 
+GO_LD_EXTRAFLAGS := -X github.com/uor-framework/client/cli.Version="$(shell git tag | sort -V | tail -1)"
```

Comment:
Is there an interest to also include the git commit which is common i tooling in related tooling?

Reply:
I can add that. That information will be useful using a build from the branch instead of the pre-built binaries.

Reply:
Added a few useful fields like platform, go version, and build date as well.